datahub-web-react/src/app/shared/tags/TagProfileDrawer.tsx

```diff
@@ -0,0 +1,417 @@
+        setDisplayColorPicker(!displayColorPicker);
+        closePicker();
+    };
+
+    const handleColorChange = (color: any) => {
+        setColorValue(color?.hex);
+    };
+
+    const EditButton = (
+        <>
+            <EditIcon twoToneColor="#52c41a" onClick={() => setShowEditModal(true)} />
+            <EditDescriptionText> Edit Description</EditDescriptionText>
+        </>
+    );
+
+    const onCloseEditModal = () => setShowEditModal(false);
+
+    // Update Description
+    const updateDescriptionValue = (value: string) => {
+        setUpdatedDescription(value);
+        return updateDescription({
+            variables: {
+                input: {
+                    description: value,
+                    resourceUrn: urn,
+                },
+            },
+        });
+    };
+
+    // Handle save button click on Edit description Modal
+    const handleSaveDescription = async (value: string) => {
+        message.loading({ content: 'Saving...' });
+        try {
+            await updateDescriptionValue(value);
+            message.destroy();
+            message.success({ content: 'Description Updated', duration: 2 });
+            onCloseEditModal();
+        } catch (e: unknown) {
+            message.destroy();
+            if (e instanceof Error) {
+                message.error({ content: `Failed to update description: \n ${e.message || ''}`, duration: 2 });
+            }
+        }
+        refetch?.();
+    };
+
+    return (
+        <>
+            <Drawer
+                width={500}
+                placement="right"
+                closable={false}
+                onClose={closeTagProfileDrawer}
+                visible={tagProfileDrawerVisible}
+                footer={
+                    <Space>
+                        <Button type="text" onClick={closeTagProfileDrawer}>
+                            Close
+                        </Button>
+                    </Space>
+                }
+            >
+                <>
+                    {/* Tag Title */}
+                    <div>
+                        <TitleLabel>Tag</TitleLabel>
+                        <ColorPicker>
+                            <ColorPickerButton style={{ backgroundColor: colorValue }} onClick={handlePickerClick} />
+                        </ColorPicker>
+                        {displayColorPicker && (
+                            <ColorPickerPopOver>
+                                <ChromePicker color={colorValue} onChange={handleColorChange} />
+                            </ColorPickerPopOver>
+                        )}
+                        <TitleText>{data?.tag?.properties?.name}</TitleText>
+                    </div>
+                    <Divider />
+                    {/* Tag Description */}
+                    {description.length === 0 ? (
+                        <>
+                            <NoDescriptionText>No Description</NoDescriptionText>
+                            {EditButton}
```

Comment:
the spacing on this edit button looks a bit off from the video. Can you give it a bit more space between description and edit button?

Reply:
+1